src/main/java/io/tiledb/java/api/Condition.java

```diff
@@ -0,0 +1,88 @@
+package io.tiledb.java.api;
+
+import io.tiledb.libtiledb.*;
+import java.lang.reflect.Array;
+import java.math.BigInteger;
+
+public class Condition implements AutoCloseable {
```

Comment:
Please validate the name of the class with @ihnorton I think python/c++/go use `QueryCondition` so we might want to keep the same naming scheme

Reply:
Done